spyderlib/config.py

```diff
@@ -169,6 +169,7 @@ def is_ubuntu():
               'animated_docks': True,
               'prompt_on_exit': False,
               'panes_locked': True,
+              'interface_language': load_lang_conf(),

```

Comment:
Another example of things we don't add are the run configuration options per file :-)